tools/python_tools/cuvslam_tools/dataset_preparation/euroc/cli.py

```
@@ -17,7 +17,7 @@
 import argparse
 import subprocess
 from pathlib import Path
-from typing import Optional
+from typing import List, Optional
```

Ruff (0.16.1)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)